sentry/src/main/java/io/sentry/UpdateStatus.java

```diff
@@ -0,0 +1,44 @@
+package io.sentry;
+
+import org.jetbrains.annotations.NotNull;
+
+/** Represents the result of checking for app updates. */
+public abstract class UpdateStatus {
```

Comment:
wondering if this can be used as a sealed class in Kotlin (i.e. doing a `when` over this)? I guess not, but this API is fine for me - just brainstorming if we should change it an enum to support `when`-statements (but then it wouldn't be easily typed)